content/docs/user-guide/appendix/cvars/_index.md

```diff
@@ -1,27 +1,56 @@
 ---
-title: CVARs
+title: Console variables (CVARs)
 ---
 
-Console variables (CVARs) can be used to set various editor and runtime options in O3DE. O3DE ships with many built-in CVARs. 
+Console variables (CVARs) can be used to set various editor and runtime options in O3DE. O3DE ships with many built-in CVARs but also supports options for configuring and grouping your own.
 
 ## Working with CVARs
-You can define your own CVARs in code using the [CVAR macro functions](/docs/user-guide/programming/az-console/#console-variables-cvars).
+
+You can define your own CVARs in code using the [CVAR macro functions](/docs/user-guide/programming/az-console/#console-variables-cvars). You can also group CVARs together in [console variable groups](/docs/user-guide/editor/console-cvars-commands/#console-variable-groups), which allow you to change multiple CVARs at once. 
 
 You can invoke CVARs in the Editor via the [console](/docs/user-guide/editor/console/).
 
 You can also pass CVARs on the command line when launching O3DE binaries via `--{cvar name} {value}` or `--{cvar name}={value}`. For example, `ServerLauncher.exe --sv_somebool=true`.
 
+Additionally, you can add CVARs to [configuration files](/docs/user-guide/editor/console/#configuring-console-variables-in-configuration-files).
+
+### Discovering CVARs
+
+The fastest way to find all the CVARs settable in the current project is via the [console variables window](/docs/user-guide/editor/console/#viewing-the-console-window). Bringing up the console window will list all the CVARs in the current project as well as allow you to set their values.
+
+You can also begin to type the start of CVAR name in the console and then use `tab` to show autocomplete and look at all the potential matches. For example, after typing `ed_` you can use the tab key to cycle through all the CVARs that impact the editor.
+
+![Automatic CVAR expansion in the console](/images/user-guide/appendix/cvars/console_autoexpand.png)
+
+### CVAR console functions
+
+There are built-in console commands that make working with CVARs easier:
+
+| Name             | Description                                                                                                                                                   |
+|------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| DumpCommandsVars | (Legacy) Will dump all the CVARs and their values to a file called `consolecommandsandvars.txt`.<br/>Prefer to use the console window to see CVARs and values | 
```

Comment:
When I run this  command on my development build, the file is written to `./Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/consolevars.txt` even when the DCCSI Gem is not active in the project. Can users specify where to write this file?

Reply:
I believe its based on your project path, will have to check.

Reply:
You can't set the location, its based on the current directory of the process. For builds, its the build folder ie build\windows_vs2019\bin\profile etc.

Reply:
The code doesn't take a path or directory, it just says write the file 'consolevars.txt' without any pathing ie

```
azfopen(&f0, "consolecommandsandvars.txt", "w");
```

Platform OS versions of fopen/fopen_s will be invoked and these set relative paths relative to the current working directory to the process invoked.

From a documentation POV, theres nothing you can currently set to control pathing. So I would suggest requesting this as a feature/reporting it as a bug, but theres nothing we can add to these docs right now.